catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -93,6 +93,7 @@
 import org.slf4j.LoggerFactory;
 import spark.Request;
 import spark.Response;
+import spark.utils.StringUtils;
```

Comment:
❓ Curious why we're using these `StringUtils` and not Apache commons?

Reply:
that was an accident; fixed